### PR TITLE
Update vw_access_aid.sql

### PR DESCRIPTION
**Pull request by sjib**

### CODE REVIEW

**Comment by sjib** [comment]
Update vw_ Tables: Error in vw_reach : Line 87
   , RE.progression_3d_geometry instead of , NEW.progression_3d_geometry - working on it

File: view/vw_wastewater_node.sql [hunk 1]
```diff
@@ -84,9 +89,9 @@ UPDATE qgep.od_wastewater_networkelement
   SET
        identifier = NEW.identifier
      , remark = NEW.remark
-     , dataowner = NEW.dataowner
-     , provider = NEW.provider
-     , last_modification = NEW.last_modification
+           , dataowner = NEW.dataowner
+           , provider = NEW.provider
```
Comment (m-kuhn) [inline view/vw_wastewater_node.sql]: Are these columns still in the game?

**Comment by m-kuhn** [comment]
Merged

File: view/vw_wastewater_node.sql (hunk 1 above)
Comment (sjib) [inline view/vw_wastewater_node.sql]: Yes, that is still there at the moment ( I know we have the fk_ now) – when we import or export data we will need a space to store it

**Pull request closed without merging**